### ISSUE TITLE
Scanning near water

### ISSUE THREAD

**Issue by daejincho** [seikur0/PGO-mapscan-opt#51]
I stay near a big lake and scanning takes forever because there are no pokemon on water. I assume after the first run, it will go a lot faster because it will skip the non-responding coordinates.

I'm thinking to grab all of these bad points on map and store them somewhere and modify the codes, so the program doesn't go through these areas every time I run it. Can somebody guide me in the right direction for this modification?

This is going to be a long list of coordinates, since the lake is about 2 miles long and has a bad shape to it.

![emptylocs](https://cloud.githubusercontent.com/assets/20749499/17318616/2cc99514-584d-11e6-99bf-d6a932483eae.png)

EDIT: I thought this was happening because of the water but this also happens at other areas. I have both python 2.7 and 3.x installed. Could this be why this is happening?

**Comment by daejincho** [comment]
Also, 'ctrl + c' used to quit the program. On this newest version, it doesn't quit but seems to 'refresh' the scan if it's stuck at some point. Btw, it gets stuck and doesn't do anything randomly throughout the scanning process and pressing ctrl+c it will force it to move on.

Is there a way to quit the program while it's running?

**Comment by seikur0** [comment]
Water cells are mostly empty, but even in cities you have some empty cells and in rural areas you have many of them as well.

I'll modify it, so Strg+C will quit it again, I wasn't aware of that. (Had this run in a compiler, which force killed it.

Blacklisting empty coordinates is a good idea, I can implement that. I feared that it might remove a cell, that isn't actually one. I tested and tweaked this a lot though and this never happened to me with the current setup.
